app/templates/base.html

```diff
@@ -15,38 +15,41 @@
 
 <body>
     <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
-        <div class="container-fluid">
-            <div class="d-flex flex-wrap align-items-center justify-content-center justify-content-lg-start">
-                <a href="/" class="d-flex align-items-center mb-2 mb-lg-0 text-white text-decoration-none">
-                </a>
-
-                <ul class="navbar-nav justify-content-center">
-                    <li class="nav-item"><a href='{{ url_for('index.index') }}' class="nav-link">Home</a></li>
-
-                    {% if current_user.is_authenticated %}
+        <div class="container d-flex justify-content-between">
+            <a class="navbar-brand" href="{{ url_for('index.index') }}">
+                <img src="static/images/moonbird.png" alt="" height="40" class="d-inline-block align-text-top">
```

Comment:
This src path will not work in route other than indexRoute, please change this 

```suggestion
                <img src="{{ url_for('static', filename='images/moonbird.png') }}" alt="" height="40" class="d-inline-block align-text-top">
```